milatools/cli/commands.py

```diff
@@ -561,15 +578,42 @@ def code(
     command_path = shutil.which(command)
     if not command_path:
         raise CommandNotFoundError(command)
+    qualified_node_name = qualified(node_name)
+
+    # TODO: Find the path to this file on other platforms.
+    vscode_settings_json_path = Path.home() / (
+        "AppData\\Roaming\\Code\\User\\settings.json"
+        if (sys.platform == "win32" or running_inside_WSL)
+        else "~/.config/Code/User/settings.json"
+    )
```

Comment:
The settings are in `$HOME/Library/Application\ Support/Code/User/settings.json` under MacOS.

Reply:
Neat! I'll use that when I make a PR for that.
Removed this chunk for now in fc61b941fdd5b5ed907b1f667c24a5dd1cd20c10